.github/ISSUE_TEMPLATE/use-case.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Use Case
+about: Use this template to propose an LWS use case.
+title: "[UC] <<brief description of use case>>"
+labels: triage, usecase
+
+---
+
```

Comment:
Maybe add a clarification for the mandatory status for each section (either here or at the section heading)? Or are all fields mandatory for the first draft?